### PR TITLE
PE-5495: add turbo chunk uploader

### PR DESCRIPTION
**Pull request by karlprieb**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/1sbrcfn5s09ko

### CODE REVIEW

File: lib/blocs/upload/upload_cubit.dart [hunk 1]
```diff
@@ -598,13 +599,27 @@ class UploadCubit extends Cubit<UploadState> {
 
     _activityTracker.setUploading(true);
 
+    final uploadType =
+        _uploadMethod == UploadMethod.ar ? UploadType.d2n : UploadType.turbo;
+
+    late TurboUploadType turboUploadType;
+    if (_containsLargeTurboUpload ?? false) {
```
Comment (atticusofsparta) [inline lib/blocs/upload/upload_cubit.dart:606]: nit:
```suggestion
    if (_containsLargeTurboUpload) {
```

**Comment by atticusofsparta** [comment]
getting 404's on transaction finalization 
image

File: lib/blocs/upload/upload_cubit.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/blocs/upload/upload_cubit.dart:606]: In Dart we can't do it. An if statement must have a non-nullable boolean.

**Comment by atticusofsparta** [comment]
Got the above issue on both prod and dev, talked with @karlprieb and we are lowering the max to 1gb which worked on both

**Review by thiagocarvalhodev — APPROVED** [review]

**Pull request closed without merging**